addons/web/static/src/js/views/graph/graph_renderer.js

```diff
@@ -1006,6 +1007,34 @@ return AbstractRenderer.extend({
         }
         return shortLabel;
     },
+    /**
+     * sort datapoints as per given order(Ascending or Descending)
+     *
+     * @private
+     * @param {Object[]} dataPoints
+     * @returns {Object[]} sorted dataPoints if orderby set on state
+     */
+    _sortDataPoints(dataPoints) {
```

Comment:
FYI, this should be done in the model, as well as _filterDataPoints. We'll do that in another PR, later on. So it's fine to keep it here for the moment.